### ISSUE TITLE
Can we support schemastore.azurewebsites.net/json

### ISSUE THREAD

**Issue by NargiT** [palantir/policy-bot#164]
Do you think it's possible to generate schema of policy-bot ?

https://schemastore.azurewebsites.net/json/

Many application do have support for it, and it works for yaml.

What do you think ?

**Comment by asvoboda** [comment]
Hey @NargiT ,

Are you interested in using this for the policy.yml file? Have you looked at https://github.com/palantir/policy-bot#testing-and-debugging-policies

**Comment by NargiT** [comment]
Yes I am using this in the CI but this request is more suited for offline development. It bring autocomplete in intellij. Renovate bot use it and it helps a lot. It can also act as documentation.
